src/assets/_sass/_dds.scss

```diff
@@ -457,21 +463,31 @@ body {
         color: $color-white;
       }
     }
+    @media(max-width:$screen-xs-min) {
+      background: url('../img/layout/team-xs.jpg') center top no-repeat;
+      background-size: cover;
+    }
   }
 
   section {
-    padding: 50px 0 0 0;
+    padding: 60px 0 0 0;
   }
 
   // Sections that are re-used on multiple parts of the site
   // For CSS that is page-specific, see below
   section.our-work {
 
     .project {
-      padding: 10px 10px 10px 10px;
-      border-top: 6px solid $background-color-secondary;
+      margin-top:30px;
```

Comment:
nit - space after the colon, here and elsewhere below

Reply:
✔️